Automattic/woocommerce-payments
# WIP: Remove ' in front of negative values on csv files

Fixes #1881 

#### Changes proposed in this Pull Request
Update the @woocommerce/components package to the latest version to remove the ' in front of negative values on csv files. As this is just an external package update I don't see any need to test the fix as there are tests on the package itself

Some tests are failing because of #2970 
Some other tests are failing because of this [issue](https://github.com/woocommerce/woocommerce-admin/issues/7703) on woocommerce-admin repository. Waiting for the merge there to continue with this PR

I Had to modify the HorizontalList component as the List component got deprecated and should be exchanged by the ExperimentalList.

Some tests snapshots became invalid as some classes and html changed after upgrading @woocommerce/components so I updated them.

**Why is this change needed?**
csv files are exported with a ' in front of negative numbers to prevent vulnerabilities. But some editors show the ' 

**What does this change do?**
This update changes de ' for a tab character which is invisible on editors

Before:
<img width="593" alt="Captura de Tela 2021-09-17 às 16 23 45" src="https://user-images.githubusercontent.com/6342964/133842959-832d8764-74a2-49d4-a977-1ff9bfc2d956.png">

After: 
<img width="592" alt="Captura de Tela 2021-09-17 às 16 24 01" src="https://user-images.githubusercontent.com/6342964/133843068-2d7dc7de-3d16-4fe2-874e-da384dc1543c.png">

#### Testing instructions

- Create an order
- Refund
- Go to Payments > Transactions inside wp-admin
- Click on Download
- Open the csv (I used Numbers)
- Make sure the ' does not appear in front of negative values

-------------------


**Post merge**
